takeaway-challenge.rb

```diff
@@ -0,0 +1,38 @@
+require './lib/dish'
+require './lib/menu'
+require './lib/menufactory'
+require './lib/orderfactory'
+require './lib/order'
+require './lib/invoice'
+require './twilio'
+
+
+
+menus = {
+  :meatliquor => [{:name=>:nigiri_sushi, :price=>13.99},
+                  {:name=>:tonkotsu_ramen, :price=>9.95},
```

Comment:
Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)